lib/templates/template-consultations.yml

```
@@ -8,10 +8,48 @@ organization:
     - es
   force_users_to_authenticate_before_access_organization: false
 scopes:
-  - code: type
+  - code: apartat
+    name: 
+      ca: Apartat
+      es: Apartado
+    scopes:
+      - code: 3actualitzacions
+        name: 
+          ca: Actualitzacions Internes
+          es: Actualizaciones Internas
+      - code: 9Altres
+          name: 
+            ca: Altres
+            es: Otros
+      - code: 1pressupost
+          name: 
+            ca: Pressupost
+            es: Presupuesto
+      - code: 0rendiciocomptes
+          name: 
+            ca: Rendició de Comptes
+            es: Rendición de Cuentas
```

```suggestion
      - code: 9Altres
        name: 
          ca: Altres
          es: Otros
      - code: 1pressupost
        name: 
          ca: Pressupost
          es: Presupuesto
      - code: 0rendiciocomptes
        name: 
          ca: Rendició de Comptes
          es: Rendición de Cuentas
```
Careful, identation!

---

Thanks, done.